homeassistant/components/tedee/__init__.py

```diff
@@ -20,12 +35,58 @@
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Integration setup."""
 
+    if CONF_WEBHOOK_ID not in entry.data:
+        new_data = entry.data.copy()
+        new_data[CONF_WEBHOOK_ID] = webhook_generate_id()
+        hass.config_entries.async_update_entry(entry, data=new_data)
```

Comment:
This was in the Withings integration because there could be a possibility that there were config entries without the webhook id set. Since this isn't released yet, maybe we can put this in the config flow so everyone has it?